File: Difficulty: Basic/Min and Max in Array/min-and-max-in-array.java
class Solution {
    public ArrayList<Integer> getMinMax(int[] arr) {
        // code Here
        ArrayList<Integer> result = new ArrayList<Integer>();
        int length = arr.length;
        int largest = arr[0];
        int smallest = arr[0];
        for(int i=1;i<length;i++){
            if(largest<=arr[i]){
                largest=arr[i];
            }
            if(smallest>=arr[i]){
                smallest=arr[i];
            }
            
        }
        result.add(smallest);
        result.add(largest);
       
        return result;
    }
}

File: Difficulty: Basic/Min and Max in Array/min-and-max-in-array.cpp
class Solution {
  public:
    vector<int> getMinMax(vector<int> &arr) {
        // code here
        vector<int> result;
        int  length = arr.size();
        int max = arr[0];
        int min = arr[0];
        for(int i=1;i<length;i++){
            if(arr[i]>max){
                max=arr[i];
            }
            if(arr[i]<min){
                min=arr[i];
            }
        
        }
        result.push_back(min);
        result.push_back(max);
        
        return result;
        
    }
};